tests/test_basic.py

```diff
@@ -64,7 +64,7 @@ def test_file_creation(rook_cluster):
     time.sleep(5)
 
     logger.debug("Mount myfs again and output the contents")
-    result = rook_cluster.kubernetes.execute_in_pod_by_label("""
+    rc, stdout, stderr = rook_cluster.kubernetes.execute_in_pod_by_label("""
```

Comment:
Could you mention (in the commit message) which commit (or at least PR) did break it?